packages/sandbox/src/index.ts

```diff
@@ -144,7 +144,10 @@ export const createTypeScriptSandbox = (
 
   const language = languageType(config)
   const filePath = createFileUri(config, compilerOptions, monaco)
-  const element = (config as any).elementToAppend ? (config as any).elementToAppend : document.getElementById(config.domID)
+  const element = "elementToAppend" in config ? config.elementToAppend : document.getElementById(config.domID)
```

Comment:
Why the switch to `in` here?

Reply:
I think its nicer to narrow the type more directly; I had to stop and figure out why the cast to any was necessary and whether it implied anything was wrong.

If the other way is preferred it can be changed back.

Relatedly, it's a bit awkward that `config.domID` is always at least an empty string due to the default value, meaning that if the ternary was flipped to `(config as any).domID ? ...`  the logic would be wrong. The same applies to my version of course, but I think the potential for error is more obvious there.